### PR TITLE
fix: Make week start on Monday instead of Sunday

### PR DESCRIPTION
**Pull request by johan-bjareholt**
Solves #180

### CODE REVIEW

**Comment by ErikBjare** [comment]
Why such a convoluted fix? Looks bug prone to me... 

Couldn't you just have had a `timeperiod === 'week'? 'isoWeek' : timeperiod`? I don't think you should use `this.timeperiod` in setDate.

File: src/views/activity/daily/ActivityDaily.vue [hunk 1]
```diff
@@ -181,6 +181,21 @@ export default {
     periodReadable: function() {
       return moment(this.timeperiod.start).format(this.dateformat);
     },
+    periodLengthMoment: function() {
```
Comment (ErikBjare) [inline src/views/activity/daily/ActivityDaily.vue:184]: This function seems redundant. It basically just does `s/week/isoWeek`.

File: src/views/activity/daily/ActivityDaily.vue [hunk 2]
```diff
@@ -209,7 +224,7 @@ export default {
     },
     setDate: function(date, periodLength) {
       const new_date = moment(date)
-        .startOf(periodLength)
+        .startOf(this.periodLengthMoment)
```
Comment (ErikBjare) [inline src/views/activity/daily/ActivityDaily.vue:227]: Don't use `this.periodLengthMoment` as it relies on `this.periodLength`, which makes it not respect the function parameter.

**Comment by johan-bjareholt** [comment]
@ErikBjare First attempt was to actually have "Day" with a capital letter as lower case letter looks odd, but that had other side effects so I dropped that. But as I never actually fixed that your solution is shorter, but it has the flaw of not reporting an error when an invalid periodLength is set (which is possible when someone changes the periodLength in the URL).

Also, I was about to remove periodLength completely from setDate as it's pointless to pass it as an argument, but obviously forgot to do that.

**Pull request merged**